domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -392,20 +456,58 @@ public Future<Void> closeClient() {
     return clientToClose.close();
   }
 
+  public Future<Void> closeClient(){
+    if (client == null){
+      return Future.succeededFuture();
+    }
+
+    PgPool clientToClose = client;
+    PgPool readClientToClose = client == readClient ? null : readClient;
+    client = null;
+    readClient = null;
+
+    closeClient(clientToClose);
+    return closeClient(readClientToClose);
+    /*
+    return CompositeFuture.all(closeClient(clientToClose), closeClient(readClientToClose))
+        .mapEmpty();
+    */
```

Comment:
Commented out? What's wrong with it? The compose suggested before is just sequental and here "concurrent".

Reply:
Removed